ledger-core/v2/insolar/payload/proto_vnvn.r0.proto

```diff
@@ -0,0 +1,192 @@
+syntax = "proto3";
+
+package payload;
+
+import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+import "github.com/insolar/assured-ledger/ledger-core/v2/insproto/ins.proto";
+
+option (insproto.notation_all) = true;
+option (insproto.zeroable_all) = true;
+option (gogoproto.goproto_enum_prefix_all) = false;
```

Comment:
if you use it by default, then it may be good to set it by default in proto-gen-ins

Reply:
@cyraxred I guess it will be better to do it in another issue. Too many changes here)